### PR TITLE
Fix #1913: Disable the auth token button when user has not activated the account

### LINKED ISSUE #1913: Disable auth token when account not activated
### Observed Behaviour
When a user is not activated , he is able to go to `Get your auth token` tab. Where it also downloads the json file with false message.

### Expected Behaviour
The tab should be disabled.

screenshot from 2018-11-19 01-01-05

@deshraj @RishabhJain2018

### PR DESCRIPTION
**Pull request by yashdusing**
Fix #1913 : Disables the auth token when user account is not activated

### CODE REVIEW

**Comment by yashdusing** [comment]
@RishabhJain2018 @deshraj Could you please verify the PR ?

**Comment by gautamjajoo** [comment]
Hey @yashdusing ! I was trying to test the PR but it seems the button is not visible when the user is not verified. Please see the attached screenshot.
screenshot from 2019-02-15 16-41-29

**Comment by yashdusing** [comment]
@gautamjajoo The pr said I should disable the ‘get you auth token’ button. So, when the user is not activated, he won’t see the button. Isn’t that the task ?

**Comment by gautamjajoo** [comment]
@yashdusing You could disable as it is done in `Show Analysis` button.

screenshot from 2019-02-15 17-05-33

**Comment by yashdusing** [comment]
@gautamjajoo Please review again. I changed the condition

**Comment by yashdusing** [comment]
@RishabhJain2018 please review

File: frontend/src/js/controllers/profileCtrl.js [hunk 1]
```diff
@@ -20,6 +20,7 @@
         vm.inputType = 'password';
         vm.status = 'Show';
         vm.token = '';
+        vm.visibleAuthButton = true;
```
Comment (RishabhJain2018) [inline frontend/src/js/controllers/profileCtrl.js]: `vm.visibleAuthButton` -> `vm.disbleAuthTokenButton`

File: frontend/src/js/controllers/profileCtrl.js [hunk 2]
```diff
@@ -44,6 +45,8 @@
         parameters.token = userKey;
         parameters.callback = {
             onSuccess: function(response) {
+                //If next call finished first and returned false
+                vm.visibleAuthButton = true && vm.visibleAuthButton;
```
[the remark this answers is not in the document]
Comment (yashdusing) [reply frontend/src/js/controllers/profileCtrl.js]: If I remember correctly, I did this to control asynchronous function calls. So, if function 2 was executed before 1, the whole thing wouldn't break. I'll have to look at the code to explain it properly

**Pull request closed without merging**